files/templates/mbta/login/resources/js/form-registration.js

```diff
@@ -0,0 +1,38 @@
+/* Registration Form Script */
+
+//window.onload = function() {
+//  setTimeout(
+//    function() {
+//      $("#token").val('this must be empty').delay(2000).queue(function() {
+//				$("#token").val((new Date).getFullYear());
+//				});
+//		
+//    }, 2000);
+//}
```

Comment:
suggestion: you don't need to check in commented-out code. if it's needed in the future, it should be possible to get it from the Git repository.